doc/source/whatsnew/v1.0.1.rst

```diff
@@ -67,7 +67,7 @@ Interval
 
 Indexing
 ^^^^^^^^
-
+- Bug in :class:`DataFrame` setting values with a slice (e.g. ``df[-4:] = 1``) indexing by label instead of position (:issue:`31469`)
```

Comment:
```suggestion
- Fixed regression in :class:`DataFrame` setting values with a slice (e.g. ``df[-4:] = 1``) indexing by label instead of position (:issue:`31469`)
```